### PR TITLE
mock-oauth-server: set OwnerClaim and DeviceIDClaim to JWT for

### PR DESCRIPTION
**Pull request by jkralik**
clientCrendetial flow

### CODE REVIEW

File: test/oauth-server/service/token.go [hunk 1]
```diff
@@ -57,8 +56,14 @@ func makeAccessToken(clientID, host, deviceID, scopes string, issuedAt, expires
 		return nil, fmt.Errorf("failed to set %v: %w", jwt.IssuerKey, err)
 	}
 	if deviceID != "" {
-		if err := token.Set(TokenDeviceID, deviceID); err != nil {
-			return nil, fmt.Errorf("failed to set %v: %w", TokenDeviceID, err)
+		if err := token.Set(uri.DeviceIDClaimKey, deviceID); err != nil {
+			return nil, fmt.Errorf("failed to set %v: %w", deviceID, err)
```
Comment (Danielius1922) [inline test/oauth-server/service/token.go]: instead uri.DeviceIDClaimKey of deviceID, key will tell you more than a uuid; or maybe log both

File: test/oauth-server/service/token.go [hunk 2]
```diff
@@ -57,8 +56,14 @@ func makeAccessToken(clientID, host, deviceID, scopes string, issuedAt, expires
...
+		}
+	}
+	if owner != "" {
+		// mock oauth server always set DeviceUserID, because it supports only one user
+		if err := token.Set(uri.OwnerClaimKey, DeviceUserID); err != nil {
+			return nil, fmt.Errorf("failed to set %v: %w", owner, err)
```
Comment (Danielius1922) [inline test/oauth-server/service/token.go]: same as above

Comment (jkralik) [reply test/oauth-server/service/token.go]: fixed

File: test/oauth-server/service/token.go (hunk 1 above)
Comment (jkralik) [reply test/oauth-server/service/token.go]: fixed

**Review by Danielius1922 — APPROVED** [review]

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**